types/braintree-web/index.d.ts

```diff
@@ -685,8 +686,8 @@ declare namespace braintree {
      *   }
      * }, callback);
      */
-     create(options: { client: Client, fields: HostedFieldFieldOptions, styles?: any }): Promise<HostedFields>;
-     create(options: { client: Client, fields: HostedFieldFieldOptions, styles?: any }, callback: callback): void;
+     create(options: { client: Client, authorization?: string, fields: HostedFieldFieldOptions, styles?: any }): Promise<HostedFields>;
+     create(options: { client: Client, authorization?: string, fields: HostedFieldFieldOptions, styles?: any }, callback: callback): void;
```

Comment:
client should be optional. quote: 

> options.authorization A tokenizationKey or clientToken. Can be used in place of `options.client`.

Reply:
in this braintree example you can how both secure 3 d and hosted fields can be used by specifying only authorization without the client https://codepen.io/braintree/pen/ezvymm

Reply:
@jbojcic1 thx for the review, amended!
I obviously missed this in the docs I've linked myself 🤦‍♂